packages/powerapps-lib/.npmignore

```diff
@@ -0,0 +1,17 @@
+# Logs
+logs
+*.log
+npm-debug.log*
+lerna-debug.log*
+
+# Dependency directories
+node_modules/
+
+# Ignore any hidden files/folders (starting with a .)
+.*
+
+# Test sources
+**/__tests__/
+**/__mocks__/
+**/*.spec.js
+**/*.test.js
```

Comment:
Should we add package-lock.json files here?

Reply:
See - https://stackoverflow.com/questions/44546718/should-package-lock-json-also-be-published
I don't think it will be published